src/Ng/Jobs/Monitoring2MonitoringJob.cs

```diff
@@ -42,6 +46,12 @@ protected override void Init(IDictionary<string, string> arguments, Cancellation
 
         protected override async Task RunInternal(CancellationToken cancellationToken)
         {
+            var currentMessageCount = await _queue.GetMessageCount(cancellationToken);
+            if (currentMessageCount > _maxRequeueQueueSize)
+            {
+                Logger.LogInformation("Can't requeue any invalid packages because the queue has too many messages!");
```

Comment:
Shouldn't this return early?

Reply:
lol yup, fixed

Reply:
may be log this as warning.. so that we can easily look it up in the logs?

Reply:
I don't think it counts as a warning-- the fact that the queue has a lot of packages doesn't signify anything is wrong with the system. We can also already find the message very easily in the logs by looking at the message directly.

Reply:
It *may* indicate something wrong, right? If so, I would make it a warning.

Reply:
Filtering is far easier on warnings than on information and like @loic-sharma said it could possibly be an issue(like it happened this week)